internal/client/job.go

```diff
@@ -423,3 +498,13 @@ func (c *Project) queueAndStreamJob(
 // to accomidate the additional time before the job was picked up when testing in
 // local Docker.
 const stateEventPause = 3000 * time.Millisecond
+
+var (
+	warnGitDirty = strings.TrimSpace(`
+       There are local changes that do not match the remote repository. By default, 
+       Waypoint will perform this operation using a remote runner that will use the 
+       remote repository’s git ref and not these local changes. For these changes 
+       to be used for future operations, either commit and push, or run the operation 
+       locally with the -local flag.
+	`)
```

Comment:
To fix the indentation here that you see in the CLI, just move everything flush with the line start. If you search for `= strings.TrimSpace(` you'll see that's what we do elsewhere.
```suggestion
There are local changes that do not match the remote repository. By default, 
Waypoint will perform this operation using a remote runner that will use the 
remote repository’s git ref and not these local changes. For these changes 
to be used for future operations, either commit and push, or run the operation 
locally with the -local flag.
`)
```